src/services/adherence.service.ts

```diff
@@ -142,8 +142,39 @@ async function getAdherenceAlerts(
   return result.data
 }
 
+async function markAdherenceAlertsAsRead(
```

Comment:
I would have a single function markAdherenceAlerts that takes read/unread as a parameter